spec/popup/format-spec.js

```diff
@@ -0,0 +1,51 @@
+import format from '../../src/common/popup/utils/format.js';
+
+describe('format util', () => {
+  const ticket = {
+    id: 'BTC-042',
+    title: 'Weird Bug while eating cake',
+    type: 'bug'
+  };
+
+  describe('commit', () => {
+    it('formats', () => {
+      const formatted = format.commit(ticket);
+      expect(formatted).toBe(`[#${ticket.id}] ${ticket.title}`);
+    });
+  });
+
+  describe('branch', () => {
+    it('formats', () => {
+      const formatted = format.branch(ticket);
+      expect(formatted)
+        .toBe(`${ticket.type || 'feature'}/${ticket.id}-${format.normalize(ticket.title)}`);
```

Comment:
Get rid of the ` || 'feature'` here. Looks like this is copied from the implementation.